app/src/main/java/ch/hikemate/app/ui/components/HikeCard.kt

```diff
@@ -73,7 +73,10 @@ fun HikeCard(
     styleProperties: HikeCardStyleProperties = HikeCardStyleProperties(),
     showGraph: Boolean = true,
 ) {
-  val displayMessage = !elevationData.isNullOrEmpty() && messageContent != null
+  val displayMessage =
+      messageContent != null &&
+          styleProperties.messageColor != null &&
+          styleProperties.messageIcon != null
```

Comment:
Question: Why do we have to enforce the `messageColor` and `messageIcon` to be non-null to display the message? Can't we display it even without it?

If you choose to keep it, could you document it?

Reply:
One needs the three of those to actually display the message. The user can pass a null message and valid icon/color, or a null icon and a valid message/color, and so on.

So I enforce we have all three to make sure we can actually display the message. Does that sound reasonable?

Reply:
Done 👌🏻